pages/product-listing/product-listing.css

```diff
@@ -0,0 +1,78 @@
+/* shared classes  */
+.color-dark-grey {
+  color: var(--dark);
+}
+.align-items-center {
+  align-items: center;
+}
+
+/* PRODUCTS PAGE CONTAINER  */
+.products {
+  display: grid;
+  grid-template-rows: 6rem minmax(0, 1fr);
+  /* max-height: 100vh; */
+}
+
+/* HEADER  */
+
+/* Navbar */
+
+.login-btn {
+  background-color: var(--primary);
+}
+
+.nav-header {
+  grid-column: 1/-1;
+}
+
+/* MAIN  */
+main {
+  display: grid;
+  height: calc(100vh - 6rem);
+  grid-template-columns: minmax(0, 1fr) minmax(0, 5fr);
+  grid-template-rows: repeat(minmax(0, 1fr));
+  column-gap: 1rem;
+}
+
+/* Sidebar of main  */
+.product-list-sidebar {
+  background-color: white;
+  box-shadow: 0.5px 0px 5px 0.1px;
+  overflow-y: scroll;
+}
+.product-list-sidebar::-webkit-scrollbar {
+  display: none;
+}
+
+.product-list-sidebar > h4 > span {
+  text-decoration: underline;
+  color: black;
```

Comment:
suggestion - 
-text-underline-offset : 2px; for  clear button

also align items center for h4 tag would look good